openapi/features-api.yaml

```diff
@@ -0,0 +1,1166 @@
+
+            Note: When using Place IDs or Saved Location IDs a maximum of 3 IDs may be used.
+
+            Note: When using lat/lon+radius, the radius must be in the format <radius><radius_unit>, where <radius> is an integer or a float number up to 2 decimal places and <radius_unit> is one of:
+            **Fields:**
+            - `m` - meters
+            - `km` - kilometers
+            - `ft` - feet
+            - `mi` - miles
+            E.g. using Place IDs:
+            ```json
+                {
+                  "location": {
+                    "place_id": [
+                      5224323,
+                      5811704,
+                      4887398
+                    ]
+                  }
+                }
+            ```
+            E.g. using Saved Location IDs:
+            ```json
+                {
+                  "location": {
+                    "saved_location_id": [
+                      "BN7ZSw8xza9FviPVfyCycd",
+                      "X3uyTFbDOUaX2q_Qh5i31b",
+                      "X3uyTFbDOUhrfq_Qh5i31A"
+                    ]
+                  }
+                }
+            ```
+            E.g. using a latitude/longitude and radius (recommended):
+            ```json
+                {
+                  "location": {
+                    "geo": {
+                      "lat": 41.75038,
+                      "lon": -71.49978,
+                      "radius": "2.62mi"
+                    }
+                  }
+                }
+            ```
+          $ref: '#/components/schemas/Location'
+        interval:
+          description: |
+            Aggregation interval.
+
+            Possible values:
+                **Fields:**
+            - `day` (default) for daily aggregation
+            - `week` for weekly aggregation
+          $ref: '#/components/schemas/Interval'
+        week_start_day:
+          description: |
+            The weekday to be treated as the start of the week.
+
+            Possible values:
+                **Fields:**
+            - `monday (default)`
+            - `tuesday`
+            - `wednesday`
+            - `thursday`
+            - `friday`
+            - `saturday`
+            - `sunday`
+
+            Only applicable when interval is set to week.
+          $ref: '#/components/schemas/Weekday'
+        predicted_events:
+          description: |
+            Include or Exclude predicted events.
+
+            Supports the following fields:
+               **Fields:**
+                - `exclude` Possible values:
+                    - `true`
+                    - `false` (default) Include predicted events
+            E.g.
+            ```json
+            {
+              "predicted_events": {
+                "exclude": false
+              }
+            }
+            ```
+          $ref: '#/components/schemas/PredictedEvents'
+        phq_attendance_academic_graduation:
```

Comment:
All `phq_attendance_*` features can be either bool or `StatsField`. Both the below are valid:

```
{
  "phq_attendance_sports": {
    "stats": ["count", "std_dev", "median"],
    "phq_rank": { 
      "gt": 50
    }
  }
}
```

```
{
  "phq_attendance_concerts": true
}
```